tools/cabana/binaryview.cc

```diff
@@ -234,6 +237,7 @@ void BinaryItemDelegate::paint(QPainter *painter, const QStyleOptionViewItem &op
 
   if (index.column() == 8) {
     painter->setFont(hex_font);
+    painter->fillRect(option.rect, item->bg_color);
```

Comment:
how about add some margins to fillrect ? e.g. 5?

Reply:
Tried that, but like it better without the gap between the colored squared.